index.html

```diff
@@ -7,14 +7,181 @@
+                    
+                        <label for="lname">Last Name</label>
+                        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+                    
+                        <label for="country">Country</label>
+                        <select id="country" name="country">
+                          <option value="australia">Australia</option>
+                          <option value="canada">Canada</option>
+                          <option value="usa">USA</option>
+                        </select>
+                    
+                        <label for="subject">Subject</label>
+                        <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+                    
+                        <input type="submit" value="Submit">
+                      </form>
+                    </div>
+                </div>
+            </div>
+        
+   <!-- MAP -->        
+            <div class="row-3">
+                <div class="col-12 md-col-6">
+                    <div class="card">
+                        <img src="/images/map.PNG" alt="hand">
+                    </div>   
+                </div> 
+                <div class="col-12 md-col-6"> 
+                <div class="row">
+          
+                        <div class="col-6">
+                            <h4>Address: </h4><h5> <br> Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis sunt placeat at odit, architecto corrupti nemo molestiae! Modi vero eius amet. Laborum assumenda, ullam beatae facere ea enim harum voluptatum.</h5>
+                        </div>
+                    <div class="col-6">
+                        <h5>Lorem ipsum dolor sit amet.</h5>
+                    </div>
+                    <div class="col-6">
+                        <hr><h4>Address: </h4><h5><br> Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis sunt placeat at odit, architecto corrupti nemo molestiae! Modi vero eius amet. Laborum assumenda, ullam beatae facere ea enim harum voluptatum.</h5>
+                    </div>
+                        <div class="col-6">
+                            <h4>jdf lorem10 dsfd df sdf dfdsf dfs sdf s s sdf sdfs fsd fs </h45>
+                        </div>
+                    </div>
+                </div>
+            </div>
+    </div>
+</section>
 
+<section>
+    <div class="container">
+        <section class= "footer">
+        <div class="row">
+                <div class="col-12 md-col-4">
+                    <div class="content">
+                        <h3>By:</h3> 
+                        <span class="copy">&copy; Abdullah Celen</span>
+                    </div>
+                </div>
+                <div class="col-12 md-col-4">
+                    <div class="content">
+                        <h3>Join us on</h3>
+                        <p>plant <a href="#"></a> <br>garden <a href="#"></a></p>
```

Comment:
You've got a couple of empty `a` elements here - if you're not going to use them, please remove them so that only used html elements are part of your site